lib/exercises.rb

```diff
@@ -1,19 +1,57 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: a lot
+# Space Complexity: O(n)
 
 def grouped_anagrams(strings)
-  raise NotImplementedError, "Method hasn't been implemented yet!"
+  hash = {}
+
+  strings.each do | string | 
+
+   a_key = string.chars.sort
+
+   if hash.key? (a_key)
```

Comment:
Just a bit more compact

```suggestion
   if hash[a_key]
```